IPython/html/static/base/less/page.less

```diff
@@ -5,16 +5,23 @@
  */
 
 
+html, body {
+    min-height: 100%;
+    height: 100%;
+    overflow: hidden; // Prevents scrollbar in html element
```

Comment:
I just checked that restoring the absolute positioning below does indeed make this block entirely unnecessary, and it should be removed.